WooCommerce/Classes/Analytics/WooAnalyticsEvent.swift

```diff
@@ -86,6 +98,14 @@ extension WooAnalyticsEvent {
     static func featureFeedbackBanner(context: FeedbackContext, action: FeatureFeedbackBannerAction) -> WooAnalyticsEvent {
         WooAnalyticsEvent(statName: .featureFeedbackBanner, properties: ["context": context.rawValue, "action": action.rawValue])
     }
+
+    static func shipmentTrackingMenu(action: ShipmentTrackingMenuAction) -> WooAnalyticsEvent {
+        WooAnalyticsEvent(statName: .shipmentTrackingMenuAction, properties: ["action": action.rawValue])
+    }
+
+    static func shippingLabelsAPIRequest(result: ShippingLabelsAPIRequestResult) -> WooAnalyticsEvent {
+        WooAnalyticsEvent(statName: .shippingLabelsAPIRequest, properties: ["action": result.rawValue])
```

Comment:
I proposed to rename `action` to `result` in p91TBi-3oZ-p2#comment-2769 - will update when we have a decision!

Reply:
Per answer in p91TBi-3oZ-p2#comment-2777, we're keeping the `action` key and I also updated the event name to be consistent with Android in: https://github.com/woocommerce/woocommerce-ios/pull/3410/commits/a2af63732eaf755ec0f925de8c652729bfdefb79